website/content/en/docs/ansible/development-tips.md

```diff
@@ -6,107 +6,115 @@ weight: 12
+This Ansible collection allows a developer to either leverage their existing
+Kubernetes resource files (written in YAML) or express the lifecycle
+management in native Ansible. One of the biggest benefits of using Ansible in
+conjunction with existing Kubernetes resource files is the ability to use
+Jinja templating so that you can customize deployments with the simplicity of
+a few variables in Ansible.
 
-The easiest way to get started is to install the modules on your local machine
-and test them using a playbook.
+The easiest way to get started is to install the collection on your local
+machine and test them using a playbook.
 
-### Installing the k8s Ansible modules
+### Installing the Kubernetes Collection for Ansible
+
+To install the Kubernetes Collection, one must first install Ansible 2.9+.
+For example, on Fedora/Centos:
 
-To install the k8s Ansible modules, one must first install Ansible 2.9+. On
-Fedora/Centos:
 ```bash
 $ sudo dnf install ansible
 ```
 
 In addition to Ansible, a user must install the [OpenShift Restclient
-Python][openshift_restclient_python] package. This can be installed from pip:
+Python](https://github.com/openshift/openshift-restclient-python) package:
+
 ```bash
 $ pip3 install openshift
 ```
 
-Finally, a user must install the Ansible Kubernetes collection from ansible-galaxy:
+Finally, install the Kubernetes Collection from ansible-galaxy:
+
 ```bash
 $ ansible-galaxy collection install community.kubernetes
 ```
 
-Alternatively, if you've already initialized your operator, you will have a `requirements.yml`
-file at the top level of your project. This file specifies Ansible dependencies that
-need to be installed for your operator to function. By default it will install the
-`community.kubernetes` collection, which are used to interact with the Kubernetes API, as well
-as the `operator_sdk.util` collection, which provides modules and plugins for operator-specific
-operations. To install the Ansible modules from this file, run
+Alternatively, if you've already initialized your operator, you may have a
+`requirements.yml` file at the top level of your project. This file specifies
+Ansible dependencies that need to be installed for your operator to function.
+By default it will install the `community.kubernetes` collection as well as
+the `operator_sdk.util` collection, which provides modules and plugins for
+operator-specific operations.
+
+To install the dependent modules from this file, run:
+
 ```bash
 $ ansible-galaxy collection install -r requirements.yml
 ```
 
-### Testing the k8s Ansible modules locally
+### Testing the Kubernetes Collection locally
 
 Sometimes it is beneficial for a developer to run the Ansible code from their
 local machine as opposed to running/rebuilding the operator each time. To do
 this, initialize a new project:
-```bash
+
+```none
 $ operator-sdk new --type ansible --kind Foo --api-version foo.example.com/v1alpha1 foo-operator
```

Comment:
Could we replace here for the Memcached one?

Reply:
See above comment.
If you feel strongly about switching to memcached example, I can revise as well.

Reply:
See the quick start: https://github.com/operator-framework/operator-sdk/blob/master/website/content/en/docs/ansible/quickstart.md

My suggestion here is: 

```suggestion
$ operator-sdk new memcached-operator --api-version=cache.example.com/v1alpha1 --kind=Memcached --type=ansible
```

Which means explain how the collections work based on in the example that has been provided in the docs already.

Reply:
I think that would be nice we keep using the same example of the quick start. however, the Memcached is not used now and it should not be a block to move forward with your improvements.